### DISCUSSION TITLE
What client app do you use?

### DISCUSSION THREAD

**Discussion by tdzz1102** [navidrome/navidrome#2375] [General]
I saw navidrome supports Airsonic and Subsonic api, which means there are too many apps to choose from, but most of them seem to be out of time. Are there some recommended options?

**Comment by dweymouth** [comment]
I use [Supersonic](https://github.com/dweymouth/supersonic) on desktop (I'm the developer :). And on iOS I use iSub (only FOSS iOS client that has gapless playback that I'm aware of). Though I do 99% of my listening on desktop while working. For Android, Symfonium and DSub are popular.

**Reply by SamuelLeitner** [reply]
Supersonic works really well, thx

**Reply by chrisisthegreat** [reply]
I use Supersonic too. I have been happy with it on Windows 11. Thanks @dweymouth!!

**Comment by ERisEPR** [comment]
For iOS I have been liking Substreamer.  The developer also has a container that will put a similar UI in a browser if you wanted that as well.

**Comment by metalheim** [comment]
I use:
Sonixd on desktop
Symfonium on Android

**Comment by sergio-ns** [comment]
Substreamer on Android

**Comment by ilPeo** [comment]
Play:Sub on iOS, even if I'm experiencing some random crashes since the last update

**Comment by codeinabox** [comment]
I've become a fan of [Tempo](https://github.com/CappielloAntonio/tempo) on Android, showing a lot of potential and open-source too

**Comment by PassiveLemon** [comment]
I've had some good results with Amperfy on IOS.
For desktop, I've also had good results with Sonixd, Feishin (Sonixd rewrite) and TauonBox.

**Comment by ilPeo** [comment]
I tried Amperfy some time ago and found it good but a little too minimal for me.I ended using Play:Sub, worth the price Il giorno 12 nov 2023, alle ore 17:47, PassiveLemon ***@***.***> ha scritto:﻿
I've had some good results with Amperfy on IOS.
For desktop, I've also had good results with Sonixd, Feishin (Sonixd rewrite) and TauonBox.

**Comment by mattneub** [comment]
Please see now https://github.com/navidrome/navidrome/discussions/3876 also.

**Comment by tannisroot** [comment]
Feishin on Desktop, Symfonium on the phone (Android)

**Comment by benjamindaines** [comment]
Symfonium

**Reply by candideu** [reply]
This is what I've settled on as well, and it's well-worth the fee! Tried Tempo, but it crashed on me, refused to open, and I had to reset everything.

On web, I use the Navidrome web app with the Spotlify-like theme. Haven't settled on a desktop app yet, but waiting on the Feishin rewrite.

**Comment by ariesps** [comment]
Audioling on Windows Desktop and Arpeggi (Testflight) on iPhone, both in development, both, the best I've tried, especially Arpeggi.

**Reply by Metal-Snake** [reply]
Arpeggi looks promising, but still has major flaws which make it unuseable for me. Like when you are in the list of songs and want to just shuffle play them all, it will only shuffle through the first n songs. So if it's sorted by artist it will play only songs by artists which start with A. 
I reported it a while ago, and also found it was already posted about on reddit and the dev commented they would fix it, but it's still broken. https://old.reddit.com/r/arpeggiApp/comments/1howrt4/feedback_for_v020_82/m4dmqoe/

**Comment by pyxfluff** [comment]
Feishin is my favorite on desktop, Amperfy on iOS seems to be the one that works best (substreamer was kinda a mess)

**Comment by mariusdmm** [comment]
Substreamer on Android
Feishin on Windows10

**Comment by Th3K1ngP1n** [comment]
Symfonium on Android. I can highly recommend it. Endless customization,  great support, looks beautiful and most of all feels light weight and snappy.

**Comment by JWdeBokx** [comment]
Indeed I also use Symfonium. It's a paid app, but it's worth it.
No subscription, just one time pay.

As free app Utlrasonic is in use by me so now and then

**Reply by Th3K1ngP1n** [reply]
Worth every penny. Monthly updates and a dev that actually implements feature requests.

**Comment by mstgrv** [comment]
On Mac, Feishin is a great experience. True gapless playback is possible when using the MPV player backend.

On iOS, unfortunately there is no perfect solution in my experience. I've tried several clients and they all have unique issues.

- Play:Sub: when playing back Opus (`.opus` or `.ogg` files) or AAC (`.m4a` files) downloaded using `yt-dlp`, you can't skip through these files during playback. If you try to do so, they'll just start playing from the start. You can kinda get it working if you let the app fully buffer one of these files, then force quit the app and re-open it - at this point the app should load with the file ready to play, and at this point you can skip through these files. 

- Amperfy: this app doesn't have true gapless playback. When listening to FLAC albums that need gapless playback, there is a tiny gap / glitch between the tracks. This app also doesn't support .ogg / Opus media (you get an error message when playing back this media type)

- Music Stream: this app also doesn't support true gapless playback. It does support .ogg / Opus files but there is a glitch similar to Play:Sub - sometimes if you pause playback (manually, or by taking out your airpod), when playback resumes it will skip back to the beginning. However this doesn't happen 100% of the time

**Reply by jessienab** [reply]
It's so frustrating that there doesn't seem to be a perfect solution for Opus/Vorbis playback AND gapless on iOS...

**Comment by dcihon** [comment]
On Linux you can use Strawberry on the Desktop . it has a plugin for subsonic

**Comment by Suman2023** [comment]
Navsonic (android)

**Comment by candideu** [comment]
There's a new app out there called [Musly](https://musly.devid.ink/) which has some potential:

<img width="467" height="934" alt="scrcpy_iZ9QmlVDT1" src="https://github.com/user-attachments/assets/ad18bcb3-0721-4441-a49c-d36f78976dbe" />

https://github.com/dddevid/Musly

**Reply by JWdeBokx** [reply]
Thanks, installed it and will try it

**Comment by dcihon** [comment]
So what do I need to do to compile it for IOS?

**Comment by elijahgronda** [comment]
Old thread but I thought I would add, for iOS look at [Arpeggi](https://www.reddit.com/r/arpeggiApp/s/T7fQGw4Kod). It is still in TestFlight, but so far it has been the absolute best for me. Very responsive devs as well.

**Comment by dcihon** [comment]
How do I try it out. I have TestFlight.   
  
Thanks
  
  
  
  
  
  
  

  
  
>

**Reply by elijahgronda** [reply]
Hey! You can join the beta using [this link](https://testflight.apple.com/join/LDWqgjAs). Try opening it in your iOS device, it takes you right to the invitation.

**Comment by acealone** [comment]
Arpeggi or Nautiline are both great choices for iOS

**Comment by nenadjokic** [comment]
Developer here, disclosing that since I mention my own app. 
I use my own app ;) for the Apple side specifically, NaviBeat covers iPhone, iPad, Mac, Apple TV, Apple Watch and CarPlay from one purchase, with the Watch doing offline playback on its own (one time purchase 5.99$). Linux version just got released this night ;) and Linux is free!

Sharing NaviBeat only for the case where someone wants the same client on the TV, macOS and the Watch as on the phone, which is the gap I built it for.

P.S. Amperfy is the free and open-source pick and a very good one, and Sonora is nice too.

**Reply by counterpaul** [reply]
I have to say, NaviBeat gets better and better with each update and so far it's the only client I've found that can reliably do gapless playback over AirPlay. If that's important to you, and you're an Apple person, NaviBeat is very worth checking out.

**Reply by nenadjokic** [reply]
Thank you, that means a lot, and gapless is the thing I am most careful about breaking, so it stays that way.

A few things have changed since I posted above. The Linux build now installs from apt and dnf repositories as well as the AppImage, so it updates with everything else on the machine, and it is still free: https://github.com/nenadjokic/navibeat-linux is where the releases and the repository instructions live, and it is also where to open an issue if something misbehaves. The Android app is in testing on Play, invite only for now, so anyone who wants in can send me the Google account they use and I will add it. The Apple side is unchanged, one purchase for iPhone, iPad, Mac, Apple TV and the Watch.

Developer, disclosing again since I am talking about my own app.

**Comment by rodleviton** [comment]
Developer here too, disclosing that up front — I built [EKO](https://github.com/reactivepixels/eko) for the Mac side specifically.

Wanted something that actually plays FLAC/hi-res bit-perfect (native decode at the file's own sample rate, no OS mixer resample) with a real signal-path indicator so you can see when it's not bit-perfect rather than just trusting a label. Talks to Navidrome/OpenSubsonic (multi-server) plus local files, gapless, ReplayGain, 10-band EQ, media keys/Now Playing, lyrics, scrobbling. It's free and MIT-licensed, no account needed.

Still early and would genuinely love feedback from people running Navidrome day to day — releases [here](https://github.com/reactivepixels/eko/releases/latest) if anyone wants to kick the tires.

**Comment by rocketguedes** [comment]
[Feishin](https://github.com/jeffvli/feishin) (`Web` / `Desktop`)
[Arpeggi](https://github.com/argie-w/Arpeggi-App) (`iOS`)

**Comment by zak-raindog** [comment]
Developer here, disclosing that up front since I'm about to name my own app.

I use **Spindle** on iOS against my own Navidrome. Plenty of good clients in this thread already,
so rather than claim mine is better, here's the specific thing it does differently:

**It streams `format=raw` by default on Wi-Fi** — the actual file at its own rate, not a
transcode. That's about 1 GB an hour for 96/24, so cellular is a *separate* setting capped at 320
by default and Low Data Mode overrides both. And because "no resampling" is easy to claim and hard
to verify, there's a test that walks the whole corpus asserting the rate the engine decoded matches
the `samplingRate` Navidrome reported — silent resampling being the bug you can't hear.

Albums pin for offline, so the library can be bigger than the phone, and the remote UI doesn't
appear at all until a server is configured.

Honest limits: **no server playlists, no starred sync** yet, and remote browsing is albums /
artists / genres / search. Tested against three Navidromes including the public demo; it uses plain
Subsonic endpoints so other servers should work, but I can't claim what I haven't run.

https://apps.apple.com/app/id6785612847 — happy to answer anything.

**Reply by Metal-Snake** [reply]
> I use **Spindle** on iOS against my own Navidrome. 
> 
> https://apps.apple.com/app/id6785612847 — happy to answer anything.

It's not available in Germany?

**Reply by zak-raindog** [reply]
Getting that sorted now. Initial launch was US but only because I'm new to iOS releases and didn't click through the right things :D

**Reply by zak-raindog** [reply]
Should be good now (or in morning), let me know if not!

**Comment by dcihon** [comment]
Is this a pay app?
  
  
  
  
  
  
  

  
  
>

**Reply by zak-raindog** [reply]
It's free to try for a week, then $6.99 to unlock permanently

**Comment by acealone** [comment]
If you don't mind sideloading on iOS you should checkout https://github.com/acealone/swiftdrome-app. Gapless and Crossfade are a key feature. And it hat listening pets as a gimmick 🫣

**Comment by manueljpy** [comment]
I made a free open-source Android DJ app and since I cannot add streaming services like Spotify or Apple music to it, I made it compatible with Subsonic so you can use your own music server.

Github: https://github.com/manueljpy/SideDeck

RIght now it lets you browse playlists and search for songs, then you can download the songs you want to your device. I'm looking into streaming the songs instead of having to download them first.

Any feedback would be appreciated!
